catalog/core/catalog-core-eventcommands/src/main/java/ddf/catalog/pubsub/command/ListCommand.java

```diff
@@ -69,10 +74,18 @@ protected Object doExecute() throws Exception {
             console.println();
             console.print(CYAN_CONSOLE_COLOR);
             console.print(SUBSCRIPTION_ID_COLUMN_HEADER);
+            console.print("\t\t| ");
+            console.print(CALLBACK_COLUMN_HEADER);
             console.println(DEFAULT_CONSOLE_COLOR);
 
-            for (String subscriptionId : subscriptionIds) {
-                console.println(subscriptionId);
+            for (String subscriptionId : subscriptionIds.keySet()) {
```

Comment:
Use an `entrySet()` instead here, as you're getting the `ServiceReference` in the loop.